### PR TITLE
Add VSCode extensions

### PR DESCRIPTION
**Pull request by thuongdv**
Add helpful VSCode extensions.

### CODE REVIEW

**Comment by thuongdv** [comment]
This is a comment

**Comment by thuongdv** [comment]
This is the second comment

**Comment by thuongdv** [comment]
> This is the second comment

this is a comment of 2nd comment.

File: README.md [hunk 1]
```diff
@@ -85,3 +85,8 @@ npx eslint .
 ```sh
 npx prettier . --write
 ```
+
+# VS Code extensions
```
Comment (thuongdangagest) [inline README.md]: Should use full name: Visual Studio Code

Comment (thuongdv) [reply README.md]: Updated.
https://github.com/thuongdv/cypress-tdd/pull/2/commits/d01092da02d8b15496d2c85731f0a74ac0ffc443

**Comment by thuongdangagest** [comment]
I got it. Thank you.
Please let me know if you would like to make any update.

---

わかった。 ありがとう。
更新をご希望の場合はお知らせください。

**Pull request merged**